pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/dao/kit/KitDaoImpl.java

```diff
@@ -76,9 +77,7 @@ public class KitDaoImpl implements KitDao {
             + "WHERE "
             + "dsm_kit_id = ?";
 
-    private static final String INSERT_KIT_TRACKING = "INSERT INTO "
-            + "ddp_kit_tracking "
-            + "SET "
+    private static final String INSERT_KIT_TRACKING = "INSERT INTO ddp_kit_tracking SET "
```

Comment:
Neat!  I was not familiar with the `insert into ... set...` syntax, but I see the benefit.